README.md

```diff
@@ -1,4 +1,8 @@
 # Py-Wpg
 Platform for assembly, annotation and visualization of bacterial genomes
 
-Exercise 1: A python (or biopython) script to query NCBI to download E coli contigs in fasta format. We download 1000 contigs and calculate N50 sitatistics 
+Exercise 1: A python (or biopython) script to query NCBI to download E coli contigs in fasta format.
+Three arguments need to be specified for running the script: your valid email address, your NCBI api key and number of assemblies you want to download.
+If you dont have an NCBI api key, create an account at https://www.ncbi.nlm.nih.gov/account/ using your email. Once you sign in, go to the top right corner where your email id displayed. Click on it and you will be directed to a new page where you can find your api key. Copy the key to use it as an argument for the script. 
+
+Usage: python fetch_contigs.py "Valid email id" "NCBI api_key" "number of assemblies" 
```

Comment:
This usage needs to be updated with the additional cmd line arguments you added

Reply:
Done